elements/rh-footer/rh-footer-block.css

```diff
@@ -15,13 +15,13 @@
 
 ::slotted(*) {
   color: #fff;
-  font-size: 14px;
+  font-size: 0.875em;
```

Comment:
We're replacing all px values to `ems` for a11y.

Reply:
Grab this from tokens?

Reply:
I think we need to break this out as a discussion.  The issue is that currently we can't use `rems` on RHDC.  RHDC sets an odd root font-size that will mess with all of our `rem` value tokens like so:

![Screen Shot 2022-06-28 at 2 31 50 PM](https://user-images.githubusercontent.com/3428964/176258894-07fefee9-e892-453e-97ba-c2dd6de61c44.png)

Reply:
Moved to the Design Tokens discussion: https://github.com/RedHat-UX/red-hat-design-system/discussions/385#discussioncomment-3042873